1_network.yml

```diff
@@ -23,40 +23,39 @@ targets:
      - 1_network/out/network.rds.ind
      - 1_network/out/subseg_distance_matrix.rds.ind
 
-  # THESE DATA TARGETS ARE FOR REPRODUCIBILITY BUT NOT LINKED TO THE PIPELINE:
   # store the DRB network from Steve in the google drive folder and fetch it.
   # This first target only needed to be run one time:
   #1_network/in/delaware_stream_temp_by_segment.zip.ind:
   #  command: gd_confirm_posted(target_name)
-  1_network/in/delaware_stream_temp_by_segment.zip:
-    command: gd_get('1_network/in/delaware_stream_temp_by_segment.zip.ind')
+  #1_network/in/delaware_stream_temp_by_segment.zip:
+  #  command: gd_get('1_network/in/delaware_stream_temp_by_segment.zip.ind')
   1_network/in/delaware_stream_temp_by_segment/delaware_segments/delaware_segments.shp:
```

Comment:
I think you can delete the lines that you moved to the `getters.yml` as you did in other files for a cleaner read.

Reply:
Will do. I was too nervous about it when I first tried moving all the getters =).